serverless.yml

```diff
@@ -0,0 +1,43 @@
+service: ${env:SERVICE_NAME, "saf-lambda-function"}
+frameworkVersion: '3'
+
+provider:
+  name: aws
+  runtime: nodejs14.x
+  iam:
+    role:
+      path: ${env:IAM_ROLE_PATH, ""}
+      permissionsBoundary: ${env:IAM_ROLE_BOUNDARY, ""}
+      statements:
+        - Effect: "Allow"
+          Action:
+            - "s3:*"
+          Resource:
+            - "*"
+  environment:
+    INPUT_BUCKET: ${env:INPUT_BUCKET}
+
+functions:
+  saf:
+    timeout: ${env:TIMEOUT, 60}
+    handler: handler.saf
+    events:
+      - s3: 
+          bucket: ${env:INPUT_BUCKET}
+          event: s3:ObjectCreated:*
+          rules:
+            - prefix: ${env:INPUT_PREFIX, ""}
+    environment:
+      COMMAND_STRING: ${env:COMMAND_STRING}
+      INPUT_PREFIX: ${env:INPUT_PREFIX, ""}
+      OUTPUT_BUCKET: ${env:OUTPUT_BUCKET, env:INPUT_BUCKET}
+      OUTPUT_PREFIX: ${env:OUTPUT_PREFIX, "results/"}
+      OUTPUT_ENABLED: ${env:OUTPUT_ENABLED, "true"}
```

Comment:
Can we set the default value for `OUTPUT_ENABLED` to false so that it can be enabled only when needed?
Depending on the saf command the user exports, the `-o` option might not be supported. I got the following error for saf-cli v1.1.6 for `saf convert hdf2splunk`:
```
Unexpected arguments: -o, /tmp/TestProfile_results.json
```

Reply:
Since the `OUTPUT_*` env vars are optional, I'd assume that it's disabled by default and I can export `OUTPUT_ENABLED=true` along with all the relevant env vars if the command requires it. So my vote is to set it false. Also, I think the boolean is being passed as a string to Lambda so we can either change the condition or change how serverless.yml process the env var. I'll share the specifics on a call.